assets/src/edit-story/output/page.js

```diff
@@ -66,36 +62,30 @@ function OutputPage({ page, autoAdvance, defaultPageDuration }) {
       id={id}
       auto-advance-after={autoAdvance ? autoAdvanceAfter : undefined}
     >
-      {backgroundFullbleedElements.length > 0 && (
-        <amp-story-grid-layer template="vertical">
+      {backgroundElements.length > 0 && (
+        <amp-story-grid-layer template="vertical" style={gridLayerStyles}>
           <div className="page-background-area" style={backgroundStyles}>
-            {backgroundFullbleedElements.map((element) => (
-              <OutputElement key={'el-' + element.id} element={element} />
-            ))}
-          </div>
-        </amp-story-grid-layer>
-      )}
-
-      {backgroundNonFullbleedElements.length > 0 && (
-        <amp-story-grid-layer template="vertical" aspect-ratio={ASPECT_RATIO}>
-          <div className="page-safe-area">
-            {backgroundNonFullbleedElements.map((element) => (
+            {backgroundElements.map((element) => (
               <OutputElement key={'el-' + element.id} element={element} />
             ))}
           </div>
         </amp-story-grid-layer>
       )}
 
       {backgroundOverlay && backgroundOverlay !== OverlayType.NONE && (
-        <amp-story-grid-layer template="vertical">
+        <amp-story-grid-layer template="vertical" style={gridLayerStyles}>
           <div
             className="page-background-overlay-area"
             style={{ ...backgroundOverlayStyles }}
           />
         </amp-story-grid-layer>
       )}
 
-      <amp-story-grid-layer template="vertical" aspect-ratio={ASPECT_RATIO}>
+      <amp-story-grid-layer
```

Comment:
This is still missing the `isFill` support, but let's defer it to the follow up pull request. I guess the action item here is: let's keep #1245 open for now.